tox.ini

```diff
@@ -72,7 +72,7 @@ deps =
   setuptools-scm
   twine
 commands =
-  {toxinidir}/scripts/pypi_publish.sh
+  {toxinidir}/scripts/publish.sh
```

Comment:
@danlamanna brought up feedback that the `npm publish` shouldn't happen within a Tox command, but should live in an `npm run` script instead. It should be easy to move over, but I want to do that before merging this.

Reply:
I'll do that in a follow-up PR.